cmake/Config.cmake.in

```diff
@@ -0,0 +1,3 @@
+include(CMakeFindDependencyMacro)
+find_dependency(Threads)
+include("${CMAKE_CURRENT_LIST_DIR}/@TARGETS_EXPORT_NAME@.cmake")
```

Comment:
We should use `check_required_components` here even if there are no components:
* https://github.com/cgold-examples/fruits/blob/a4b78e26ae3a6c7244174ae869f3447f36871eb4/cmake/template/Config.cmake.in#L4
* https://gitlab.kitware.com/cmake/cmake/merge_requests/2138

Reply:
Should be good now